ch360/extractors.go

```diff
@@ -27,57 +26,67 @@ type Extractor struct {
 
 type ExtractorList []Extractor
 
-func (client *ExtractorsClient) issueRequest(ctx context.Context, method string, suffix string) (*http.Response, error) {
-	return client.issueRequestWith(ctx, method, suffix, nil, nil)
+func (client *ExtractorsClient) Create(ctx context.Context, name string, config io.Reader) error {
+	_, err := newRequest(ctx, "POST", client.baseUrl+"/extractors/"+name, config).
+		issue(client.requestSender)
+
+	return err
 }
 
-func (client *ExtractorsClient) issueRequestWith(ctx context.Context, method, suffix string, body io.Reader, headers map[string]string) (*http.Response, error) {
-	request, err := http.NewRequest(method,
-		client.baseUrl+"/extractors/"+suffix,
-		body)
+func (client *ExtractorsClient) CreateFromJson(ctx context.Context, name string, jsonTemplate io.Reader) error {
+	template, err := NewModulesTemplateFromJson(jsonTemplate)
 
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	request = request.WithContext(ctx)
+	return client.CreateFromModules(ctx, name, *template)
+}
 
-	for k, v := range headers {
-		request.Header.Add(k, v)
-	}
+type ModulesTemplate struct {
```

Comment:
💪 We've been using the term `ExtractorTemplate` for this in e.g. the extractors service, so would be helpful to maintain consistency.